### ISSUE TITLE
SI Basic 2021 does not work with BakingTray because the way the power/depth is handled seems different

### ISSUE THREAD

**Issue by raacampbell** [SainsburyWellcomeCentre/BakingTray#395]
I got an error after starting SI then BT because `hSI.hBeams.pzAdjust` did not exist. The error that appears on screen is:
```
Unrecognized method, property, or field 'pzCustom' for class 'scanimage.components.Beams'.

Error in SIBT/returnScanSettings (line 57)
    scanSettings.powerZAdjustType = obj.hC.hBeams.pzCustom; % What sort of adjustment (if empty it's default exponential)
```

**Comment by raacampbell** [comment]
Hopefully is fixed: 361b317049621b5d884b4dfc7fe4cb23ddd2d945
Both reading the settings and re-applying them have been sorted.

**Comment by raacampbell** [comment]
Seems to all work
